File: Modul_4/Bahasa_C/PRAK404-2510817120005-NorKhalisyah.c
#include <stdio.h>

int main() {
    int pilih;
    double a, b, hasil;

    printf("Pilih Program\n");
    printf("1. Penjumlahan\n");
    printf("2. Pengurangan\n");
    printf("3. Perkalian\n");
    printf("4. Pembagian\n");
    printf("5. exit\n");
    printf("Masukkan Pilihan : ");
    scanf("%d", &pilih);

    if (pilih >= 1 && pilih <= 4) {
        printf("Masukkan nilai pertama : ");
        scanf("%lf", &a);
        printf("Masukkan nilai kedua : ");
        scanf("%lf", &b);

        if (pilih == 1) hasil = a + b;
        else if (pilih == 2) hasil = a - b;
        else if (pilih == 3) hasil = a * b;
        else hasil = a / b;

        const char *ops[] = {"penjumlahan", "pengurangan", "perkalian", "pembagian"};
        printf("Hasil %s antara %.2f dengan %.2f adalah %.2f", ops[pilih - 1], a, b, hasil);
    }

    else if (pilih == 5)
        printf("Terimakasih, telah menggunakan kalkulator NOR KHALISYAH");

    else
        printf("Input anda salah, silahkan coba lagi");

    return 0;
}

File: Modul_4/Bahasa_Python/PRAK404-2510817120005-NorKhalisyah.py
print("Pilih Program")
print("1. Penjumlahan")
print("2. Pengurangan")
print("3. Perkalian")
print("4. Pembagian")
print("5. exit")

pilih = int(input("Masukkan Pilihan : "))

if 1 <= pilih <= 4:
        a = float(input("Masukkan nilai pertama : "))
        b = float(input("Masukkan nilai kedua : "))

        if pilih == 1: 
            hasil = a + b
        elif pilih == 2: 
            hasil = a - b
        elif pilih == 3: 
            hasil = a * b
        else: 
            hasil = a / b

        ops = ["penjumlahan", "pengurangan", "perkalian", "pembagian"]
        print(f"Hasil {ops[pilih - 1]} antara {a:.2f} dengan {b:.2f} adalah {hasil:.2f}")

elif pilih == 5:
        print("Terimakasih, telah menggunakan kalkulator NOR KHALISYAH")

else:
        print("Input anda salah, silahkan coba lagi")